pkg/peertls/tlsopts/tls.go

```diff
@@ -15,8 +18,14 @@ import (
 	"storj.io/storj/pkg/storj"
 )
 
+// TransportCredentials wraps an embedded credentials.TransportCredentials.
+// It decorates the `ClientHandshake` method.
```

Comment:
This comment could be better. Maybe `TransportCredentials wraps credential.TransportCredentials ensuring invalid certificates don't hang dialing.`

The information about embedding and decorating usually don't offer insight why something exists.

Reply:
:+1: